### ISSUE TITLE
Importing binding name 'EventEmitter' is not found.

### ISSUE THREAD

**Issue by hypery2k** [NathanWalker/angular-seed-advanced#270]
**IMPORTANT**: This repository's issues are reserved for feature requests and bug reports. Do not submit support requests here.

**Steps to reproduce and a minimal demo of the problem**
- done a git clone
- run `npm i`
- run `npm run start.ios`

**Current behavior**

App doesn't start:

```
Importing binding name 'EventEmitter' is not found.
```

**Expected/desired behavior**

App starts

**Other information**
- Used Node 5.12.0

**Comment by netwiser** [comment]
Same issue here. After #266 is resolved (should be), I tried this seed again. Actually I have been watching and waiting this seed to be ready for quite a long time. Unfortunately, I experienced the same issue as above.

More detail here:
file:///app/app/frameworks/i18n/services/multilingual.service.js:7:30: JS ERROR SyntaxError: Importing binding name 'EventEmitter' is not found.

There is another issue but not a show stopper:
Oct 13 20:49:34 Jamess-MacBook nativescript[1812]: objc[1812]: Class PLBuildVersion is implemented in both /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/PrivateFrameworks/AssetsLibraryServices.framework/AssetsLibraryServices (0x11f6e1910) and /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/PrivateFrameworks/PhotoLibraryServices.framework/PhotoLibraryServices (0x11f50b210). One of the two will be used. Which one is undefined.

Environment:
Fresh clone the seed git. Tried both nativescript and nativescript@next. 
MacOS, Xcode 8, Node 6.6.0, npm 3.10.3, ruby 2.3.1p112,

**Comment by NathanWalker** [comment]
@hypery2k @netwiser got a fix coming for this shortly...

**Comment by NathanWalker** [comment]
master has this fix now 👍 

After you pull, best to do this:

```
npm run clean
```

**Comment by netwiser** [comment]
Try again,
npm run clean, npm run start.ios
and re-clone the seed, npm i, npm run start.ios

Both having the same issue still
file:///app/app/frameworks/i18n/services/multilingual.service.js:7:30: JS ERROR SyntaxError: Importing binding name 'EventEmitter' is not found.

**Comment by netwiser** [comment]
Tried with nativescript 2.3.0 and 2.4.0--2016-10-13, the same.

**Comment by ajanytime** [comment]
Im having the same issue here,

1: a fresh clone
2: npm i
3: npm run start.ios

result,

Oct 13 23:00:35 AJs-MacBook-Pro nativescript[24777]: objc[24777]: Class PLBuildVersion is implemented in both /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/PrivateFrameworks/AssetsLibraryServices.framework/AssetsLibraryServices (0x11d5b7910) and /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/PrivateFrameworks/PhotoLibraryServices.framework/PhotoLibraryServices (0x11d3e1210). One of the two will be used. Which one is undefined.
Oct 13 23:00:35 AJs-MacBook-Pro nativescript[24777]: 1 0x1069a6940 -[TNSRuntime executeModule:]
Oct 13 23:00:35 AJs-MacBook-Pro nativescript[24777]: 2 0x1065546d2 main
Oct 13 23:00:35 AJs-MacBook-Pro nativescript[24777]: 3 0x10a3a768d start
Oct 13 23:00:35 AJs-MacBook-Pro nativescript[24777]: file:///app/app/frameworks/i18n/services/multilingual.service.js:21:30: JS ERROR SyntaxError: Importing binding name 'EventEmitter' is not found.
Oct 13 23:00:35 AJs-MacBook-Pro com.apple.CoreSimulator.SimDevice.84EDE544-20CB-4192-ABED-7945A0610B67.launchd_sim24587: Service exited due to Segmentation fault: 11

**Comment by NathanWalker** [comment]
Fixed a diff build issue on master. Still investigating this particular issue.

**Comment by NathanWalker** [comment]
It's coming from `ng2-translate` as best I can tell at moment, may be a new change with the recent `3.1.0`...trying to resolve tonight.

**Comment by hypery2k** [comment]
happened also with ng2-translate 3.0.0

**Comment by NathanWalker** [comment]
The specific issue reported in this issue is now fixed on master.
However another issue cropped up that I am still working through with {N} build specifically mentioned here: https://github.com/NathanWalker/angular-seed-advanced/issues/272

Sorry for this inconvenience, it is frustrating. I will have a solution for that new issue soon.

**Comment by ignaciolarranaga** [comment]
@NathanWalker any update ?, I just start facing the same after the update.

**Comment by NathanWalker** [comment]
Fixed on master. Ensure you 'rm -rf node_modules' then 'npm i'

**Comment by sunjith** [comment]
ng2-translate@2.5.0 has a dependency on angular@2.0.0
